<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityPage</class>
 <widget class="QDialog" name="ActivityPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblStatus_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Status: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="SearchGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>921</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Directory Options</string>
   </property>
   <widget class="QPushButton" name="btnEncryptSearch">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search to Encrypt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBoxLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>901</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecryptSearch">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search to Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIndicator_Encrypt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblIndicator_Decrypt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded. In mode Encrypt</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcess">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ResultsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>921</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptDecryptButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_decoded">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>381</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
